### DISCUSSION TITLE
pagination button animation

### DISCUSSION THREAD

**Discussion by ghost** [bootstrap-vue/bootstrap-vue#5867] [Q&A] [answered]
I like to add animation with animate.css to a selected and or hovered element in the pagination row. 

Sofar nothing made me implement the `v-b-hover` attribute in vue-bootstrap pagination. 
Has anyone an idea how this could be achieved ? I tried the given attributes like change and page-class etc. It works on the component element but not on the individual button, i also tried slots but no avail.

Thanks

**Comment by Hiws** [comment, marked as answer]
There's no reason to use `v-b-hover` in this case.
This can be accomplished by using normal CSS and targeting the appropriate classes.
 
In the example below i add a custom class via the `page-class` prop, to limit the animation to the numbered page buttons.
<https://codepen.io/Hiws/pen/RwRWZQB>

**Comment by ghost** [comment]
hey there @Hiws yes thanks that works too, but i had to use animate.css with three different classes.
I could solve it with this:

       ` hoverPaginate() {
            if(process.client) {
               this.$nextTick(function () {
                  const animateItems = document.getElementsByClassName("animate-paginate");

                  [].forEach.call(animateItems, a => {
                     a.addEventListener("mouseover", function(e) {
                        if(!this.classList.contains('disabled')) {
                           this.classList.add("animate__animated", "animate__rubberBand", "animate__faster");
                        }
                     });

                     a.addEventListener("mouseout", function(e) {
                        this.classList.remove("animate__animated", "animate__rubberBand", "animate__faster"); 
                     });
                  });
               });
            }
         },`

Thanks
